cli.js

```diff
@@ -57,17 +63,22 @@ const speedText = () =>
 		downloadColor(downloadSpeed());
 
 const speed = () => speedText() + '\n\n';
+const getVerboseLog = () => `Latency: ${data.latency}${data.latencyUnit} (unloaded) ${data.bufferbloat}${data.bufferbloatUnit} (loaded)\nClient: ${data.client.location}  ${data.client.ip}  ${data.client.isp}\nServer(s): ${data.serverLocations}`;
```

Comment:
This should probably be split up to several template strings, or I could add newlines and strip indent using [`strip-indent`](https://github.com/sindresorhus/strip-indent). Let me know what you think is better.